amethyst_rendy/src/camera.rs

```diff
@@ -725,6 +727,25 @@ mod tests {
         );
     }
 
+    #[test]
+    fn world_to_screen() {
+        let screen = ScreenDimensions::new(1024, 768, 1.0);
+
+        let ortho = Camera::standard_2d(screen.width(), screen.height());
+        let mut transform = Transform::default();
+
+        let center_screen = Point2::new(screen.width() / 2.0, screen.height() / 2.0);
+        let top_left = Point2::new(0.0, 0.0);
+        let bottom_right = Point2::new(screen.width() - 1.0, screen.height() - 1.0);
```

Comment:
those points are unused. Have you forgot to assert something?

Reply:
Yah, removed.